Backend/src/test/java/idealab/api/operations/PrintJobOperationsTest.java

```diff
@@ -252,4 +261,262 @@ public void getDeletablePrintJobs(){
         Assert.assertEquals(result.getPrintJobs().get(0).getId(), printJob.getId());
     }
 
+
+    @Test
+    public void createNewPrintJob() {
+        GetPrintJobResponse response = new GetPrintJobResponse();
+
+        byte[] a = hexStringToByteArray("e04fd020ea3a6910a2d808002b30309d");
+        MultipartFile file = new MockMultipartFile("Something", a);
+
+        PrintJobNewRequest request = new PrintJobNewRequest();
+        request.setColor("RED");
+        request.setComments("COMMENTS");
+        request.setCustomerFirstName("test");
+        request.setCustomerLastName("testLast");
+        request.setEmail("test@email.com");
+        request.setFile(file);
+
+        CustomerInfo customerInfo = new CustomerInfo();
+        customerInfo.setFirstName("test");
+
+        ColorType color = new ColorType();
+        color.setColor("RED");
+        color.setAvailable(true);
+
+        Employee e = new Employee();
+        e.setId(999);
+
+        EmailHash emailHash = new EmailHash();
+        emailHash.setEmailHash("test@email.com");
+
+        Queue queue = new Queue(1);
+
+        Map<String, String> data = new HashMap<>();
+        data.put("filePath", "DROPBOX_PATH");
+        data.put("sharableLink", "http://testlink.com");
+
+        PrintJob printJob = new PrintJob();
+        printJob.setColorTypeId(color);
+        printJob.setComments("COMMENTS");
+        printJob.setCreatedAt(LocalDateTime.now());
+        printJob.setDropboxPath("DROPBOX_PATH");
+        printJob.setDropboxSharableLink("http://testlink.com");
+        printJob.setId(1);
+        printJob.setUpdatedAt(LocalDateTime.now());
+        printJob.setStatus(Status.PENDING_REVIEW);
+        printJob.setEmployeeId(e);
+        printJob.setEmailHashId(emailHash);
+        printJob.setQueueId(queue);
+        printJob.setUpdatedAt(LocalDateTime.now());
+
+        List<PrintJob> printJobData = new ArrayList<>();
+        printJobData.add(printJob);
+
+
+        response.setSuccess(true);
+        response.setMessage("Successfully saved new file to database!");
+        response.setData(printJobData);
+        response.setHttpStatus(HttpStatus.ACCEPTED);
+
+        when(emailHashRepo.findByEmailHash(any())).thenReturn(emailHash);
+        when(customerInfoRepo.findByEmailHashId(any())).thenReturn(customerInfo);
+        when(colorTypeRepo.findByColor(any())).thenReturn(color);
+        when(employeeRepo.findEmployeeByUsername(any())).thenReturn(e);
+        when(printJobRepo.save(any())).thenReturn(printJob);
```

Comment:
I am fine merging these tests in as is and improving them later.   This statement in particular causes some weird interactions in the test because it short circuits a large portion of the print job creation process and no matter what the input request or data object return, the final comparison is always going to be true.

The original operation I wrote probably wasn't the easiest to test for so I see why this is required.  I will try thinking of a better way to handle this for future testing 👍